app/src/main/report/components/ReportsList/ReportListTable.tsx

```diff
@@ -1,39 +1,42 @@
 import * as React from "react";
 import {Report} from "../../../shared/models/Generated";
 import ReactTable, {Column, Filter, FilterRender, ReactTableFunction, RowRenderProps} from 'react-table'
+
 import {
-    LatestVersion,
-    latestVersionAccessorFunction,
-    LatestVersionCell,
-    versionFilterMethod,
+    AggregatedVersionCell,
+    BasicVersionDetails, getLatestVersion,
+    VersionCell,
+    aggregatedVersionFilterMethod, versionIdAccessorFunction,
     versionSortMethod
 } from "./ReportListColumns/VersionColumn";
 import {
     PublishStatusCell,
     PublishStatusFilter,
-    publishStatusFilterMethod
+    aggregatedPublishStatusFilterMethod
 } from "./ReportListColumns/PublishStatusColumn";
-import {ReportLatestVersionFilter} from "./ReportListColumns/LatestVersionFilter";
-import {nameAccessorFunction, NameCell} from "./ReportListColumns/NameColumn";
+import {ReportVersionFilter} from "./ReportListColumns/ReportVersionFilter";
+import {nameAccessorFunction} from "./ReportListColumns/NameColumn";
 
 export interface ReportsListTableProps {
     reports: Report[]
     isReviewer: boolean;
 }
 
 export interface ReportRowRenderProps extends RowRenderProps {
-    original: Report,
-    value: string | LatestVersion | boolean;
+    row: ReportRow,
+    value: string | BasicVersionDetails | boolean;
 }
 
-export interface ReportRowProps {
-    latest_version: LatestVersion,
+export interface ReportRow {
+    version: BasicVersionDetails,
     name: string,
     author: string,
     requester: string,
-    published: boolean,
+    published?: boolean,
+
+    [key: string]: string | boolean | BasicVersionDetails | ReportRow[]
 
-    [key: string]: string | boolean | LatestVersion
+    _subRows?: ReportRow[]
```

Comment:
Does this need an underscore in its name to work with the plugin? If not, it's quite out of keeping with our normal code style.

Reply:
Unfortunately I can't find another way of interacting with the sub rows, other than by this (presumably intended to be private) property.

Reply:
Figured this out, you can actually specify a `subRowsKey` property in the component props to give this a nicer name :)